src/main/java/Deadline.java

```diff
@@ -0,0 +1,29 @@
+public class Deadline extends Task{
+
+    protected String deadline;
+
+    public Deadline(String description) {
+        super(description.substring(description.indexOf("deadline") + 8, description.indexOf("/")).trim());
+        deadline = description.substring(description.indexOf("/")).replace("/by", "").trim();
+    }
+
+    @Override
+    public String getTaskType() {
+        return ("D");
+    }
+
+    @Override
+    public String getDeadline(){
+        return (deadline);
+    }
```

Comment:
Spacing between method name and '{' should be consistent

Reply:
okay! Thank you for pointing it out!